integration/test/src/docker/nginx.conf

```diff
@@ -0,0 +1,22 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+
+  sendfile on;
```

Comment:
What does this do?

Reply:
This allows nginx to send chunks of a file directly over the wire rather than copying it to a buffer first. It's a minor performance optimization that we probably don't need.

Reply:
Removed: https://github.com/cerner/beadledom/pull/149/commits/8a3c86cbe865def41f7369374be99da2c50fde2b